Makefile

```diff
@@ -0,0 +1,50 @@
+ENVS := flake8,py27,py36
+export PYTHONPATH := :$(CURDIR)/modules:$(CURDIR)/resources/lib:$(CURDIR)/test
+addon_xml := addon.xml
+
+# Collect information to build as sensible package name
+name = $(shell xmllint --xpath 'string(/addon/@id)' $(addon_xml))
+version = $(shell xmllint --xpath 'string(/addon/@version)' $(addon_xml))
+git_branch = $(shell git rev-parse --abbrev-ref HEAD)
+git_hash = $(shell git rev-parse --short HEAD)
+
+zip_name = $(name)-$(version)-$(git_branch)-$(git_hash).zip
+include_files = addon.py addon.xml LICENSE.txt modules/ README.md resources/ service.py
```

Comment:
Miss also setup.py, sorry didn't see it before.

Reply:
The setup.py is not needed (or wanted) because it is not part of the add-on, so it should not be in the ZIP file. We may even remove it at some point.

Reply:
You're right! I'm getting tired, 3:30 AM ^^ Sack time!